Dockerfile

```diff
@@ -14,17 +21,20 @@ ENV SHA="${SHA}"
 ARG BUILDTIME=unknown
 ENV BUILDTIME="${BUILDTIME}"
 
-COPY --from=py-deps /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
+#COPY --from=py-deps /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
+
+ENV PYTHONPATH="${PYTHONPATH}":/python/pkgs
+COPY --from=py-deps /python/__pypackages__/3.9/lib /python/pkgs
+COPY --from=py-deps /python/__pypackages__/3.9/bin/* /bin/
```

Comment:
I'm not sure putting this stuff in `/bin` will work, but I'm assuming you tested it - I believe you